src/main/java/net/dkamps18/channelpoints/Handler/PubSubHandler.java

```diff
@@ -47,11 +50,11 @@ public void onMessage(String message) {
 									Location loc = p.getLocation();
 									p.setVelocity(new Vector(loc.getDirection().getX() * 80, loc.getY() * 40, loc.getDirection().getZ() * 80));
 									PubSubHandler.this.plugin.disablefall.add(p.getUniqueId());
-									ResultSet dbr = PubSubHandler.this.plugin.dbu.get_user_data(p.getUniqueId().toString());
+									ResultSet dbr = PubSubHandler.this.plugin.databaseUtil.getUserData(p.getUniqueId().toString());
 									try {
 										if (dbr.next()) {
 											// See this awesome code by Marenthyu to make the patch requests proxy through https://github.com/Dkamps18/MC-ChannelPoints/issues/1
-											// PubSubHandler.this.plugin.tapi.acceptreward(rid, d.getAsJsonObject("redemption").get("id").getAsString(), dbr.getString("twitchid"), dbr.getString("authtoken"));
+											PubSubHandler.this.plugin.tapi.acceptreward(rid, d.getAsJsonObject("redemption").get("id").getAsString(), dbr.getString("twitchid"), dbr.getString("authtoken"));
```

Comment:
This was commented out due to it being for the event only shouldn't really be used was more a emergency fix and therefore commented out cause it won't be up anymore

Reply:
In the copy I saw it wasn't commented, so I just undid it.